resources/frontend_client/index_template.html

```diff
@@ -19,16 +19,18 @@
         <main class="Main flex flex-column flex-full" ng-view></main>
     </body>
 
-
-    <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js"></script>
-    <script>
-        WebFont.load({
-          google: { families: ['Lato:n3,n4,n7'] }
-        });
+    <script type="text/javascript">
+     (function loadSrc(src, onload) {
+         var script = document.createElement('script');
+         script.type  = "text/javascript";
+         script.async = true;
+         script.src   = src;
+         if (onload) script.onload = onload;
+         document.body.appendChild(script);
+         return loadSrc;
+     })('https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js', 'WebFont.load({google:{families:["Lato:n3,n4,n7"]}});')('https://maps.googleapis.com/maps/api/js', null);
```

Comment:
This seems fine, aside from some style preferences.

Most people frown upon using strings of code as event handlers, just use an anonymous function `function() { WebFont.load({google:{families:["Lato:n3,n4,n7"]}}); }`

It would also be nice if each invocation of the function were on it's own line. I'd just something like this:

```
(function() {
    function loadSrc(src, onload) {
        var script = document.createElement('script');
        script.type  = "text/javascript";
        script.async = true;
        script.src   = src;
        if (onload) script.onload = onload;
        document.body.appendChild(script);
    }
    loadSrc('https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js', function() {
        WebFont.load({google:{families:["Lato:n3,n4,n7"]}});
    });
    loadSrc('https://maps.googleapis.com/maps/api/js');
})();
```